src/PointwiseFunctions/Hydro/InitialData/IrrotationalBns.hpp

```diff
@@ -47,6 +61,20 @@ tnsr::iJ<DataType, 3> derivative_rotational_shift_over_lapse(
     const tnsr::iJ<DataType, 3>& deriv_of_shift, const Scalar<DataType>& lapse,
     const tnsr::i<DataType, 3>& deriv_of_lapse,
     const tnsr::iJ<DataType, 3>& deriv_of_spatial_rotational_killing_vector);
+/// @}
+
+/// @{
+/// \brief Compute the enthalpy density squared from other hydro variables and
+/// the spacetime
+///
+/// The eqn. is identical to B&S 15.76
+/// \f[
+///   h^2 \rho^2 = \frac{1}{\alpha^2} \left(C + B^i D_i \Phi\right)^2 - D_i
```

Comment:
Something is confusing doxygen around here, but I don't know exactly what.  You might need to switch to `/*! */` comments for multiline math.